railties/lib/rails/generators/rails/app/templates/Gemfile

```diff
@@ -21,6 +21,11 @@ source 'https://rubygems.org'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+<%- if options.api? -%>
+# Use ActiveModelSerializers to serialize JSON responses
+gem 'active_model_serializers', '~> 0.10.0.rc1'
```

Comment:
But you still need to implement 6 serializers when you have 6 models. Am I missing something?